.github/workflows/main-rendering.yml

```diff
@@ -83,6 +83,17 @@ jobs:
             cd ${{ secrets.DOC_SEARCH_PATH }}
             ./bin/console docsearch:import ${{ github.event.client_payload.type_short }}/${{ github.event.client_payload.vendor }}/${{ github.event.client_payload.name }}/${{ github.event.client_payload.target_branch_directory }}
 
+      - name: Create symlinks
+        uses: appleboy/ssh-action@master
+        with:
+          host: ${{ secrets.DEPLOY_DOCS_HOST }}
+          username: ${{ secrets.DEPLOY_DOCS_USERNAME }}
+          key: ${{ secrets.DEPLOY_KEY }}
+          script: |
+            cd ${{ secrets.TARGET_PATH }}/Home
+            ln -sn res/js/piwik.js .
```

Comment:
Thanks, I can use this as a base (the Home directory itself is a symlink, which in turn then contains nested symlinks, which is why it's a bit more complicated but it's a good starting point)

Reply:
But what exactly are you doing manually then? I would expect this should run but well, I don't have your insights :)

Reply:
Manually I'm doing absolute symlinks, I just shortened them in the issue because I wanted to avoid disclosing full server paths.